src/Command/Project/ProjectCreateCommand.php

```diff
@@ -146,14 +172,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $this->stdErr->writeln("The project is now ready!");
+        if ($template_options['initialize']) {
+            // Use the existing initialize command.
+            $project = $this->selectProject($subscription->project_id);
+            $environment = $this->api()->getEnvironment('master', $project, null, true);
+            $environment->initialize($subscription->project_options['initialize']['profile'], $subscription->project_options['initialize']['repository']);
```

Comment:
Are we certain at this point that the subscription will contain this data? always? how certain?

Reply:
I've plopped in some checks to make sure required information is coming through. I'm not wedded to the error message though, so please do review.